src/NzbDrone.Core/Download/DownloadEventHub.cs

```diff
@@ -40,6 +40,9 @@ public void Handle(DownloadCompletedEvent message)
                 !message.TrackedDownload.DownloadItem.CanBeRemoved ||
                 message.TrackedDownload.DownloadItem.Status == DownloadItemStatus.Downloading)
             {
+                var downloadClient = _downloadClientProvider.Get(message.TrackedDownload.DownloadClient);
+                downloadClient.MarkItemAsImported(message.TrackedDownload.DownloadItem);
```

Comment:
Not sure if this is exactly where this should be. I figured there is not "import marking" if the item is removed from the download client

Reply:
It's where it should be, but it's probably better to invert the if. So If (Should be removed) { Remove } else { MarkItemAsImported }

But, you should put these two lines in a separate function and add the same try catch logic as with RemoveFromDownloadClient, we don't want this to throw an exception up.
Also add the Debug line, so ppl can see in the debug log that this action is taken.

Reply:
What action exactly should be specified here. That we are "marking an item as imported" ?

Reply:
something like ba5c28b ?

Reply:
yup, fine by me.
Users will probably not understand what's happening exactly, lacking in-depth knowledge, but at least they know some post-import action was taken.